internal/test/e2e/e2e_test.go

```diff
@@ -299,7 +275,11 @@ func setupKind(t *testing.T, ctx context.Context, stopped chan struct{}) *kubect
 		configPath: kind.KubeConfigPath(),
 	}
 
-	installOperator(t, kubectl, kind, imageTar)
+	t.Log("Setting the environment variable KUBECONFIG", kubectl.configPath)
+	err := os.Setenv("KUBECONFIG", kubectl.configPath)
+	require.NoError(t, err)
```

Comment:
This is necessary so that when we call back into the Makefile, it uses the correct kube context in targets that call `kubectl`.